### DISCUSSION TITLE
Welcome to awesome-OS-clones Discussions!

### DISCUSSION THREAD

**Discussion by zriyansh** [prik-k/awesome-OS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rajaniraiyn** [comment]
Hello everyone.
I am @Rajaniraiyn. I have created this [windows11](https://github.com/Rajaniraiyn/windows11).

@prik-k thankyou so much to include mine project.

I am very eager to get some ideas and suggestions here.

**Reply by zriyansh** [reply]
You replied too early! I was just preparing to mention everyone whose project I have included! Anyways thanks a lot!

**Comment by fotang** [comment]
Thank you for the list. Just what I have spent hours googling for.

By the way, I have come across another JS desktop framework called NjDesktop (https://github.com/beerwin/njdesktop2).
